web/src/js/index.js

```diff
@@ -1,473 +1,479 @@
-  response.json().then(function (data) {
-    let env = Object.keys(data)[0];
-    let API_BASE = data[env]["urls"]["apiGatewayBaseURL"];
-
-    API_GET_ALLOWLIST = API_BASE + API_GET_ALLOWLIST;
-    API_SERVICES = API_BASE + API_SERVICES;
-    API_CRUD_ALLOWLIST = API_BASE + API_CRUD_ALLOWLIST;
-    API_EXECLOG = API_BASE + API_EXECLOG;
-
-    for (output of data[env]["outputs"]) {
-      if (output["OutputKey"] === "AccountID") {
-        app.accountId = output["OutputValue"];
-        document.title = "AWS Auto Cleanup - " + output["OutputValue"];
-        break;
-      }
-    }
-    if (API_KEY) {
-      init();
-    }
-  });
-});
+var API_GET_ALLOWLIST = "/allowlist";
+var API_SERVICES = "/settings/service";
+var API_CRUD_ALLOWLIST = "/allowlist/entry/";
+var API_EXECLOG = "/execution/";
+var API_KEY = "";
+
+// Utility functions
+function convertJsonToGet(formJSON) {
+  let formURL = "";
+  for (var key in formJSON) {
+    formURL += key + "=" + formJSON[key] + "&";
+  }
+  formURL.substr(0, formURL.length - 1);
+  return formURL;
+}
+
+// Init Vue instance
+var app = new Vue({
+  el: "#app",
+  data: {
+    accountId: "",
+    apiKey: "",
+    executionLogActionStats: {},
+    executionLogDataTables: "",
+    executionLogKey: "",
+    executionLogList: [],
+    executionLogMode: "",
+    executionLogRegionStats: {},
+    executionLogSearchTerm: "",
+    executionLogServiceStats: {},
+    executionLogTable: [],
+    resourceIdPlaceholder: "",
+    resourceList: [],
+    selectedComment: "",
+    selectedExpiration: 0,
+    selectedOwner: "",
+    selectedResource: "",
+    selectedResourceId: "",
+    selectedService: "",
+    serviceList: [],
+    serviceSettings: [],
+    serviceSettingsFlat: [],
+    showApiKeyPopup: true,
+    showExecutionLogListLoadingGif: false,
+    showExecutionLogLoadingGif: false,
+    showExecutionLogPopup: false,
+    showHelpPopup: false,
+    showAllowlistDeletePopup: false,
+    showAllowlistLoadingGif: false,
+    showAllowlistPopup: false,
+    allowlist: [],
+    allowlistDataTables: "",
+    allowlistSearchTerm: "",
+  },
+  methods: {
+    // Allowlist
+    closeAllowlistDeletePopup: function () {
+      this.selectedResourceId = "";
+      this.showAllowlistDeletePopup = false;
+    },
+    closeAllowlistInsertPopup: function () {
+      this.resourceIdPlaceholder = "";
+      this.resourceList = [];
+      this.selectedComment = "";
+      this.selectedExpiration = 0;
+      this.selectedOwner = "";
+      this.selectedResource = "";
+      this.selectedResourceId = "";
+      this.selectedService = "";
+      this.showAllowlistPopup = false;
+    },
+    createAllowlistEntry: function () {
+      let formData = {
+        resource_id:
+          this.selectedService +
+          ":" +
+          this.selectedResource +
+          ":" +
+          this.selectedResourceId,
+        owner: this.selectedOwner,
+        comment: this.selectedComment,
```

Comment:
Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/servian/aws-auto-cleanup/pullRequest?prid=9792602)
```suggestion
        comment: this.selectedComment,
```